### PR TITLE
UpgradeableAccountFactory (AA 0.7.0 & OZ 5.0.0) + SimpleERC20Paymaster + ERC20Paymaster

### PR DESCRIPTION
**Pull request by 0xFirekeeper**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -5,6 +5,9 @@
   "author": "Krishang <krishang@thirdweb.com>",
   "license": "Apache-2.0",
   "devDependencies": {},
-  "dependencies": {},
+  "dependencies": {
+    "@account-abstraction/contracts": "^0.7.0",
+    "@openzeppelin/contracts": "^5.0.0"
+  },
```
Comment (jakeloo) [inline package.json:11]: Is it possible to use `forge install` so we only use 1 package manager instead of 2?

Comment (0xFirekeeper) [reply package.json:11]: difficult in this case

Comment (0xFirekeeper) [reply package.json]: ok we're good now

**Comment by nkrishang** [comment]
Closing for now since the code is unrelated to the repo

**Pull request closed without merging**